runtime/src/lib.rs

```diff
@@ -436,6 +438,29 @@ impl orml_tokens::Trait for Runtime {
     type WeightInfo = ();
 }
 
+parameter_types! {
+    pub const ProxyDepositBase: Balance = 1;
+    pub const ProxyDepositFactor: Balance = 1;
+    pub const MaxProxies: u16 = 4;
+    pub const MaxPending: u32 = 2;
+    pub const AnnouncementDepositBase: Balance = 1;
+    pub const AnnouncementDepositFactor: Balance = 1;
+}
+impl pallet_proxy::Trait for Runtime {
+    type Event = Event;
+    type Call = Call;
+    type Currency = orml_tokens::CurrencyAdapter<Runtime, ()>;
```

Comment:
How does `()` implement the `Get<T::CurrencyId>` bound? We can set it to USDv since it's the main asset to be moving around, see how they do with the test https://github.com/open-web3-stack/open-runtime-module-library/blob/master/tokens/src/mock.rs#L123